### PR TITLE
Config option for repairing 

### PR DESCRIPTION
**Pull request by changednamedes**
added option to toggle off repair of pouches for anyone doing this before lunars

### CODE REVIEW

File: runelite-client/src/main/java/net/runelite/client/plugins/microbot/runecrafting/gotr/GotrScript.java [hunk 1]
```diff
@@ -150,13 +149,12 @@ public boolean run(GotrConfig config) {
                     if (repairCells()) return;
 
                     if (usePortal()) return;
-                    //mine huge guardian remains
```
Comment (gmason0) [inline runelite-client/src/main/java/net/runelite/client/plugins/microbot/runecrafting/gotr/GotrScript.java:153]: Why remove informational comments that the original developer placed here?

Comment (changednamedes) [reply runelite-client/src/main/java/net/runelite/client/plugins/microbot/runecrafting/gotr/GotrScript.java:153]: Refactor from copilot for some reason

Comment (gmason0) [reply runelite-client/src/main/java/net/runelite/client/plugins/microbot/runecrafting/gotr/GotrScript.java:153]: I dont see any benefit of doing this, just make things more ambiguous

Comment (changednamedes) [reply runelite-client/src/main/java/net/runelite/client/plugins/microbot/runecrafting/gotr/GotrScript.java:153]: As I said, wasn't intentional - refactor from copilot did it.. I'll add them back

**Comment by chsami** [comment]
We already have a check inside npcContact to see if the player has lunar or not.

Rs2Magic.java

```
public static boolean npcContact(String npcName) {
        if (!isLunar()) {
            Microbot.log("Tried casting npcContact, but lunar spellbook was not found.");
            return false;
        }
        final int chooseCharacterWidgetId = 4915200;
        boolean didCast = cast(MagicAction.NPC_CONTACT);
        if (!didCast) return false;
        boolean result = sleepUntilTrue(() -> Rs2Widget.getWidget(chooseCharacterWidgetId) != null && !Rs2Widget.isHidden(chooseCharacterWidgetId), 100, 5000);
        if (!result) return false;
        boolean clickResult = Rs2Widget.clickWidget(npcName, Optional.of(75), 0, false);
        if (!clickResult) return false;
        Rs2Player.waitForAnimation();
        return true;
    }
```

**Comment by changednamedes** [comment]
> We already have a check inside npcContact to see if the player has lunar or not.
> 
> Rs2Magic.java
> 
> ```
> public static boolean npcContact(String npcName) {
>         if (!isLunar()) {
>             Microbot.log("Tried casting npcContact, but lunar spellbook was not found.");
>             return false;
>         }
>         final int chooseCharacterWidgetId = 4915200;
>         boolean didCast = cast(MagicAction.NPC_CONTACT);
>         if (!didCast) return false;
>         boolean result = sleepUntilTrue(() -> Rs2Widget.getWidget(chooseCharacterWidgetId) != null && !Rs2Widget.isHidden(chooseCharacterWidgetId), 100, 5000);
>         if (!result) return false;
>         boolean clickResult = Rs2Widget.clickWidget(npcName, Optional.of(75), 0, false);
>         if (!clickResult) return false;
>         Rs2Player.waitForAnimation();
>         return true;
>     }
> ```

Yeah but it spams it over and over in GOTR if you don't have it.. this way it doesn't actually call the function so doesn't spam the chat

**Pull request closed without merging**